qemu/tests/qmp_command.py

```diff
@@ -251,29 +214,45 @@ def send_cmd(cmd):
     except Exception as err:
         test.fail(err)
 
+    # sleep 10s to make device_del take effect
+    if qmp_cmd == "device_del id=device_del01; netdev_del id=netdev_del01":
```

Comment:
I am not sure here would `('device_del' in qmp_cmd) or ('netdev_del' in qmp_cmd)`  be better? I suppose what you care is the command, you don't care the device name and better not make it so specific.

Reply:
make sense,   this is used to match case 'qmp_netdev_del'.
Just confirmed,  case `qmp_device_del`  also use `device_del` in `qmp_cmd`.  And `netdev_del` only used by case `qmp_netdev_del`.  Will update with `'netdev_del' in qmp_cmd`,  Thanks a lot !

Reply:
@vivianQizhu   updated, please help review again, thanks a lot !